src/index.html

```diff
@@ -6,8 +6,64 @@
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="shortcut icon"
+      href="images/star-active.svg"
+      type="image/x-icon"
+    >
   </head>
+
   <body>
-    <h1>Stars</h1>
+    <main class="containet">
+      <!-- <h1>Stars</h1> -->
+
+      <ul class="stars stars--0">
+        <il class="stars__star"></il>
```

Comment:
```suggestion
        <li class="stars__star"></li>
```

There are no `<il>` tag in html